src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts

```diff
@@ -0,0 +1,41 @@
+/** Angular imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+
+/**
+ * Individual collection sheet component.
+ */
+@Component({
+  selector: 'mifosx-individual-collection-sheet',
+  templateUrl: './individual-collection-sheet.component.html',
+  styleUrls: ['./individual-collection-sheet.component.scss']
+})
+export class IndividualCollectionSheetComponent implements OnInit {
+    /** Collection sheet form. */
```

Comment:
variables are not aligned, indent properly

Reply:
Done